src/org/zaproxy/zap/GuiBootstrap.java

```diff
@@ -71,13 +73,6 @@
 
     private final Logger logger = Logger.getLogger(GuiBootstrap.class);
 
-    /**
-     * Flag that indicates whether or not the look and feel was already set.
-     * 
-     * @see #setupLookAndFeel()
-     */
-    private boolean lookAndFeelSet;
```

Comment:
Is this no longer necessary? (Seems to be setting the L&F twice when starting for the first time.)

Reply:
Nowhere in the function setupLookAndFeel, I have used this variable.

Reply:
Yeah, it was removed from `setupLookAndFeel`, but seems to be necessary.

Reply:
@thc202 L&F is set twice because the code at https://github.com/zaproxy/zaproxy/blob/develop/src/org/zaproxy/zap/GuiBootstrap.java#L367,L388 first sets L&F to system L&F and then switches to Nimbus L&F.

Reply:
I was not referring to the execution of `setLookAndFeel` but `setupLookAndFeel`, now is being done twice when ZAP starts for the first time (while before it wasn't).

Reply:
@thc202 well, then I will revert this change. I will set variable as true whenever L&F is set.

Reply:
Restoring the previous code/behaviour should be good.